src/plot/plot.jl

```diff
@@ -371,39 +453,16 @@ function tenders!(
     ax::Axis,
     tender_soln::Vector{HierarchicalRouting.TenderSolution}
 )::Axis
-    # Create custom colormap, skipping the first two colors (yellow and black)
-    colormap = distinguishable_colors(length(tender_soln) + 2)[3:end]
-
-    # TODO: Plot critical path (longest) thicker than other paths
-    for (t_n, t_soln) in enumerate(tender_soln)
-        base_hue = convert_rgb_to_hue(colormap[t_n])
-        s = length(t_soln.sorties)
-        palette = sequential_palette(base_hue, s + 3)[3:end]
-
-        for (sortie, color) in zip(t_soln.sorties, palette[1:s])
-            linestrings!(ax, sortie, color=color)
-        end
-    end
-    return ax
+    colormap = create_colormap(getfield.(tender_soln, :id))
+    return route!(ax, tender_soln, colormap)
 end
 function tenders!(
     ax::Axis,
     tender_soln::Vector{HierarchicalRouting.TenderSolution},
     num_clusters::Int64
 )::Axis
-    colormap = distinguishable_colors(num_clusters + 2)[3:end]
-
-    # TODO: Plot critical path (longest) thicker than other paths
-    for t_soln in tender_soln
-        base_hue = convert_rgb_to_hue(colormap[t_soln.id])
-        s = length(t_soln.sorties)
-        palette = sequential_palette(base_hue, s + 3)[3:end]
-
-        for (sortie, color) in zip(t_soln.sorties, palette[1:s])
-            linestrings!(ax, sortie, color=color)
-        end
-    end
-    return ax
+    colormap = create_colormap(collect(1:num_clusters))
```

Comment:
We now have a method that supports unit ranges so if you feel inclined, you could that:

```suggestion
    colormap = create_colormap(1:num_clusters)
```

Reply:
Addressed by [73e5602](https://github.com/ryu-AIMS/HierarchicalRouting.jl/pull/80/commits/73e5602a049ae6d914dbd302331503f8834353af)